lib/demo/interactors/create_comment.rb

```diff
@@ -0,0 +1,23 @@
+require 'lotus/interactor'
+
+module Demo
+  class CreateComment
+    include Lotus::Interactor
+
+    def initialize(params:, repository: CommentRepository)
```

Comment:
I'd remove the repository from the constructor, and have it as an hardcoded dependency.